docs/langref/relay_pattern.rst

```diff
@@ -28,17 +28,34 @@ Such a language is not just useful for building a rewriter but also providing ex
 
 In the backend world, we could use the same machinery to build a higher level API using bring your own code generation. This API takes set of patterns describing your hardware capabilities and an external compiler, providing a relatively smooth heterogeneous experience out of the box.
 
-Examples
-========
+Pattern Examples
+================
 
-There are quite a few properties that are worth matching of operators below we examine how to match tree properties, and expand on some use cases that are not fully explored in the prototype. The first example is a simple case where we want to match one operator with a single input OR another operator with a single input, see the below diagram for a graphical representation and corresponding code::
+There are quite a few properties of operators that are worth matching. Below we examine how to match tree properties, and expand on some use cases that are not fully explored in the prototype. This section
+demonstrates how to write patterns. It is recommended to check `tests/python/relay/test_dataflow_pattern.py`_
+for more use cases.
+
+.. _tests/python/relay/test_dataflow_pattern.py: https://github.com/apache/incubator-tvm/blob/master/tests/python/relay/test_dataflow_pattern.py
+
+Matching One of Two Ops
+***********************
+
+The first example is a simple case where we want to match one operator with a single input OR another operator with a single input, see the below diagram for a graphical representation and corresponding code:
```

Comment:
there is no diagram

Reply:
@mbrookhart the diagrams are only available in the RFC but not in this document. Although we can upload the figures to web-data, I would prefer to keep the current version because I think the code itself is clear enough. Thus I removed this sentence.